src/autoscaling/snapshotter.js

```diff
@@ -13,7 +13,7 @@ const DEFAULT_OPTIONS = {
     clientSnapshotIntervalSecs: 1,
     maxUsedMemoryRatio: 0.7,
     snapshotHistorySecs: 30,
-    maxClientErrors: 1,
+    maxClientErrors: 3,
```

Comment:
We could even leave this one at 1 as we use 3rd retries.

Reply:
Let's try 3 and see what happens.